src/Json/Value.php

```diff
@@ -606,6 +606,49 @@ public function ¿dateTime(string $format = DateTime::ATOM): ?DateTimeImmutable
         return $this->dateTime($format);
     }
 
+    /**
+     * Assert this value is an int, assert it conforms to a given date time format, then return that date time.
```

Comment:
> Assert this value is a number, then return a date time measured in that many seconds since the Unix Epoch.

To remain consistent with the other documentation.